examples/security/securitycenter-pricing/main.tf

```
@@ -0,0 +1,8 @@
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_security_center_subscription_pricing" "example" {
+    tier = "Standard"
+    resource_type = "VirtualMachines"
```

Style / fmt change (as per `terraform fmt`)
```suggestion
    tier          = "Standard"
    resource_type = "VirtualMachines"
```